size mismatch in PING on openbsd

```
[23/281] Compiling C object src/irc/core/libirc_core.a.p/irc-commands.c.o
../src/irc/core/irc-commands.c:547:49: warning: format specifies type 'long' but the argument has type 'long long' [-Wformat]
        str = g_strdup_printf("%s PING %ld %ld", data, tv / G_TIME_SPAN_SECOND, tv % G_TIME_SPAN_SECOND);
                                       ~~~             ^~~~~~~~~~~~~~~~~~~~~~~
                                       %lld
../src/irc/core/irc-commands.c:547:74: warning: format specifies type 'long' but the argument has type 'long long' [-Wformat]
        str = g_strdup_printf("%s PING %ld %ld", data, tv / G_TIME_SPAN_SECOND, tv % G_TIME_SPAN_SECOND);
                                           ~~~                                  ^~~~~~~~~~~~~~~~~~~~~~~
                                           %lld
2 warnings generated.
[52/281] Compiling C object src/fe-common/irc/libfe_common_irc.a.p/fe-ctcp.c.o
../src/fe-common/irc/fe-ctcp.c:131:30: warning: format specifies type 'long *' but the argument has type 'gint64 *' (aka 'long long *') [-Wformat]
        if (sscanf(data, "%ld %ld", &tv, &tv2) < 1) {
                          ~~~       ^~~
                          %lld
../src/fe-common/irc/fe-ctcp.c:131:35: warning: format specifies type 'long *' but the argument has type 'gint64 *' (aka 'long long *') [-Wformat]
        if (sscanf(data, "%ld %ld", &tv, &tv2) < 1) {
                              ~~~        ^~~~
                              %lld
2 warnings generated.
```

reported by qbit

--- a/src/fe-common/irc/fe-ctcp.c
+++ b/src/fe-common/irc/fe-ctcp.c
@@ -128,7 +128,10 @@ static void ctcp_ping_reply(IRC_SERVER_REC *server, const char *data,
 
 	g_return_if_fail(data != NULL);
 
-	if (sscanf(data, "%ld %ld", &tv, &tv2) < 1) {
+	if (sscanf(data,
+	           "%" G_GINT64_FORMAT " "
+	           "%" G_GINT64_FORMAT,
+	           &tv, &tv2) < 1) {
 		char *tmp = g_strconcat("PING ", data, NULL);
 		ctcp_default_reply(server, tmp, nick, addr, target);
 		g_free(tmp);

--- a/src/irc/core/irc-commands.c
+++ b/src/irc/core/irc-commands.c
@@ -544,7 +544,10 @@ static void cmd_ping(const char *data, IRC_SERVER_REC *server, WI_ITEM_REC *item
 
 	tv = g_get_real_time();
 
-	str = g_strdup_printf("%s PING %ld %ld", data, tv / G_TIME_SPAN_SECOND, tv % G_TIME_SPAN_SECOND);
+	str = g_strdup_printf("%s PING "
+	                      "%" G_GINT64_FORMAT " "
+	                      "%" G_GINT64_FORMAT,
+	                      data, tv / G_TIME_SPAN_SECOND, tv % G_TIME_SPAN_SECOND);
 	signal_emit("command ctcp", 3, str, server, item);
 	g_free(str);
 }

